Python/import.c

```diff
@@ -1745,7 +1745,7 @@ PyObject *
 PyImport_Import(PyObject *module_name)
 {
     PyThreadState *tstate = _PyThreadState_GET();
-    static PyObject *silly_list = NULL;
+    PyObject *silly_list = NULL;
```

Comment:
"silly list" is not a great name, I propose to rename it to "from_list".

Also you must put a DECREF somewhere, otherwise the code will leak.